offix/src/main/java/org/aerogear/offix/Offix.kt

```diff
@@ -84,15 +73,17 @@ fun ApolloClient.enqueue(
              5. The user resolves conflicts and send the mutation back to the library.
              6. Make a recursive call to the enqueue() function and retry mutation again.
              */
-            if (response.data() == null && responseWithConflicts.isNotEmpty()) {
-                val serverClientData = getServerClientData(responseWithConflicts)
 
-                val retryMutation =
-                    responseCallback.onConflictDetected(serverClientData)
+            Log.d("Extension list Size: ", " size of  list  ${list.size}")
+            Log.d("Response DATA: ", " ${response.data()}")
+
+            if (list.isNotEmpty()) {
+
+                val retryMutationList = responseCallback.onConflictDetected(list)
```

Comment:
Let's assume that we have a single conflict for now.

Reply:
Okay, so no need of the list of mutation, right?